source/ranges.tex

```diff
@@ -6415,7 +6415,7 @@
     V @\exposid{base_}@ = V();                                          // \expos
 
     @\exposidnc{non-propagating-cache}@<remove_cv_t<@\exposidnc{InnerRng}@>> @\exposid{inner_}@;    // \expos, present only
-                                                            // when \tcode{!is_reference_v<\exposid{InnerRng}>}
+                                                            // if \tcode{!is_reference_v<\exposid{InnerRng}>} is \tcode{true}
```

Comment:
Do we have to use the negation here, or should this be "if is_reference... is false"? @CaseyCarter?

Reply:
Personally, I would prefer a "... is false" formulation without the negation.

Reply:
That seems like a reasonable simplification.

Reply:
I can't find a single "`!`bool-condition is `true`", so I think they should be the latter.